config-templates/config.php

```diff
@@ -197,6 +197,61 @@
     ),
 
 
+
+    /*
+     * Database
+     *
+     * This database configuration is optional. If you are not using
+     * core funcationlity or modules that require a database, you can
+     * skip this configuration.
+     */
+
+    /*
+     * Database connection string.
+     * Ensure that you have the required PDO database drive installed
+     * for your connection string.
+     */
+    'database.dsn' => 'mysql:host=localhost;dbname=saml',
```

Comment:
Should these options be commented out? In the end, this is optional, so those copying and pasting the template should not have database options configured (not to talk about defaults) if they are not going to use a database, IMHO.